src/fetch/period_slippage.py

```diff
@@ -19,12 +22,32 @@ def slippage_query(dune: DuneAnalytics) -> str:
     )
 
 
+@dataclass
+class SolverSlippage:
+    """Total amount reimbursed for accounting period"""
+    solver_address: Address
+    solver_name: str
+    # ETH amount (in WEI) to be deducted from Solver reimbursement
+    amount_wei: int
+
+    @classmethod
+    def from_dict(cls, obj: dict) -> SolverSlippage:
+        """Converts Dune data dict to object with types"""
+        return cls(
+            solver_address=Address(obj['solver_address']),
+            solver_name=obj['solver_name'],
+            # TODO - Reverse the signature of the value
```

Comment:
If you wanna do this, you might wanna do it in this PR.

For me, its not so important.

Reply:
Actually I will remove this todo. It's no longer necessary as I have written the code in a way that it doesn't matter. I just thought if we wanted to call things "deduct slippage" that we should be subtracting, but I went with "add slippage" so it's all good now.

Reply:
removed